core/management/commands/seed.py

```diff
@@ -32,6 +33,7 @@
 DEFAULT_NUMBER_PARTICIPANTS = 1000
 DEFAULT_NUMBER_VISITS = 100
 DEFAULT_NUMBER_INSURERS = 10
+DEFAULT_NUMBER_PROGRAM_AVAILABILITIES = 10
```

Comment:
There are about 8 programs and 5 weekdays, so 8 * 5 = 40.  But you only have 10 PROGRAM_AVAILABILITIES, so hardly any programs will be open on a given day.  You could simply increase the number, but then you might have overlapping and conflicting availabilities, e.g. CM available on Friday 8am-3pm but also 9am-4pm.  I think a more natural way to do it would be to cycle through the programs and days of the week and either have random hours available, or none at all.

Reply:
Does this parameter do anything now? I think you can cut it.